DependencyInjection/SuluFormExtension.php

```diff
@@ -72,7 +81,7 @@ public function load(array $configs, ContainerBuilder $container)
         // add dynamic lists
         foreach ($config['dynamic_lists'] as $key => $value) {
             $parameter = 'sulu_form.dynamic_lists.' . $key . '.config';
-            $container->setParameter($parameter, $value);
+            $container->setParameter($parameter, $value ?: []);
 
             $definition = new Definition(DynamicListNavigationProvider::class);
             $definition->addArgument('%' . $parameter . '%');
```

Comment:
discuss with @wachterjohannes we need to distinguish between tab name and type name so we need to add the type to the dynamic lists:

```
    dynamic_lists:
        content:
            default:
                type: page
                property: form
            campaign:
                type: page
                property: form
        article:
            article_default:
                type: article
                property: form
            article_recipe_gallery:
                type: article
                property: form
            video_default:
                type: article
                property: form
```